### DISCUSSION TITLE
BLContext setTransform & scale

### DISCUSSION THREAD

**Discussion by Sosukodo** [blend2d/blend2d#222] [General]
I've noticed that when scale is called on a BLContext followed by setTransform, I have to manually *scale to each matrix parameter. Is this by design? My expectation would be that the current scale be applied to the BLMatrix2D automatically.

**Comment by kobalicek** [comment]
Set transform replaces the user transformation matrix, if you want to combine with the current matrix, use `applyTransform()` instead.

**Reply by Sosukodo** [reply]
That makes sense. Sorry :-)
